ibmcloud/README.md

```diff
@@ -77,136 +91,65 @@ kubectl label node $node node-role.kubernetes.io/worker=
 ```
 
 ## Deploy the Confidential-containers operator
+The `caa-provisioner-cli` simplifies deploying the operator and the cloud-api-adaptor resources on to any cluster. See the [test/tools/README.md](../test/tools/README.md) for full instructions. To create an ibmcloud ready version follow these steps
+
+```
+cd test/tools
+make BUILTIN_CLOUD_PROVIDERS="ibmcloud" all
+```
+
+This will create `caa-provisioner-cli` in the `test/tools` directory. To use the tool with an existing self-managed cluster you will need to setup a `.properties` file containing the relevant ibmcloud information to enable your cluster to create and use peer-pods. Use the following commands to generate the `.properties` file, if not using a selfmanaged cluster please update the `terraform` commands with the appropriate values manually.
+
+```bash
+export IBMCLOUD_API_KEY= # your ibmcloud apikey
+export PODVM_IMAGE_ID= # the image id of the peerpod vm uploaded in the previous step
+export PODVM_IMAGE_ARCH= # the architecture of the peerpod image (amd64/s390x)
+export PODVM_INSTANCE_PROFILE= # instance profile name that runs the peerpod (bx2-2x8 or bz2-2x8 for example)
+
+cd ibmcloud/cluster # Must be run in the same directory as the terraform project
```

Comment:
We have a similar situation here - we move into `ibmcloud/cluster` but never out of it, but the next set asssumes we are in the repo directory, so another pushd/popd might be the answer